LSManager.js

```diff
@@ -0,0 +1,61 @@
+class UserPages {
+  constructor() {
+    this.userPages = []
+  }
+  addUserPage(userPage) {
+    this.userPages.push(userPage)
+  }
+  getAllPages() {
+    return this.userPages
+  }
+  clearData() {
+    this.userPages = []
+  }
+}
+
+class UserPage {
+  constructor(userData) {
+    this._userInfo = userData.userInfo
+    this._friends = userData.friends
+    this._kwQuote = userData.kwQuote
+    this._pokemon = userData.pokemon
+    this._loremIpsum = userData.loremIpsum
+  }
+}
+
+class LocalStorageManager {
+  constructor() {
+    this.isEmpty = localStorage.getItem("users") === null ? true : false
```

Comment:
Use a trinary operator when you want to perform something. 
You can rewrite this line to be shorter: ` this.isEmpty = localStorage.getItem("users") === null`